src/main/java/formflow/library/FormFlowController.java

```diff
@@ -64,12 +88,27 @@ protected FlowConfiguration getFlowConfigurationByName(String flow) {
     return flowConfigurationList.get(0);
   }
 
+  /**
+   * Checks if there are any flows matching the String parameter.
```

Comment:
I think this should be "Checks if there are any flows matching the flow name passed in."  That's more specific and easier to read.